src/ffi.rs

```diff
@@ -374,23 +333,45 @@ unsafe fn generic_free(ptr: *mut u8, heap: &crate::WrappedHeap) {
 
 /// Call this when we have an IPC IRQ. Not `extern C` as its not called by the
 /// library, only our interrupt handler code.
+// This function seems to be based on this verion in C:
+// https://github.com/NordicSemiconductor/nrfx/blob/98d6f433313a3d8dcf08dce25e744617b45aa913/drivers/src/nrfx_ipc.c#L146-L163
 pub unsafe fn nrf_ipc_irq_handler() {
     // Get the information about events that fired this interrupt
     let events_map = (*nrf9160_pac::IPC_NS::ptr()).intpend.read().bits();
 
+    #[cfg(feature = "defmt")]
+    defmt::trace!("IPC start");
```

Comment:
We don't really want to log this. It was there for just debugging but is generally useless